R/render.r

```diff
@@ -422,6 +422,11 @@ data_deps <- function(pkg, depth) {
       )
     )
   )
+
+  pkgdown_sass <- path_pkgdown("css", paste0("BS", bs_version), "pkgdown.css")
```

Comment:
Now the pkgdown css is a sass partial.

Reply:
Should we rename it to `pkgdown.sass` just to make it clear?